### PR TITLE
Commit to test gitflow

### PR DESCRIPTION
**Pull request by ejfranco18**
We created a file on views components to test the git flow.

### CODE REVIEW

File: app/views/components/_test-view.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Testing</h1>
```
Comment (nykloss) [inline app/views/components/_test-view.html.erb]: can't we add another test?

**Comment by kokodrillo** [comment]
dont like the h1

File: app/views/components/_test-view.html.erb (hunk 1 above)
Comment (nykloss) [reply app/views/components/_test-view.html.erb]: Que bonitos hojos tienes!

Comment (ejfranco18) [reply app/views/components/_test-view.html.erb]: Already made the changes

**Pull request merged**